apps/web-app/app/app.css

```
@@ -40,6 +40,21 @@
     --radius: 0.625rem;
+
+    /* Device chrome insets (0 when viewport-fit≠cover or no hardware intrusion).
+       Bottom nav height ADDS the inset so labels stay in a full 4rem content band
+       instead of border-box eating h-16 into the home-indicator padding. */
+    --safe-area-top: env(safe-area-inset-top, 0px);
+    --safe-area-right: env(safe-area-inset-right, 0px);
+    --safe-area-bottom: env(safe-area-inset-bottom, 0px);
+    --safe-area-left: env(safe-area-inset-left, 0px);
+    --mobile-bottom-nav-content-height: 4rem;
+    --mobile-bottom-nav-height: calc(
+      var(--mobile-bottom-nav-content-height) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)